.github/workflows/stale.yaml

```diff
@@ -0,0 +1,37 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 60
+          days-before-issue-close: 14
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it has not had recent activity.
+            It will be closed if no further activity occurs.
```

Comment:
nit: Perhaps we can mention it will be closed in 14 days, when we post stale issue message?